wgengine/wglog/wglog_test.go

```diff
@@ -20,7 +20,6 @@ func TestLogger(t *testing.T) {
 	}{
 		{"hi", "hi", false},
 		{"Routine: starting", "", true},
-		{"peer(IMTB…r7lM) says it misses you", "[IMTBr] says it misses you", false},
```

Comment:
why's this test deleted? isn't this line the whole point of why this thing existed in the first place?

Reply:
It's hard to figure out how to test it. We don't have any way of constructing a `*device.Peer`. We'd have to build an entire wireguard setup, at which point the test is sensitive to any changes in wireguard-go's logging.

An alternative approach is to look for any `fmt.Stringer` instead of any `*device.Peer`, call `String()` on it, do the lookup, and then replace if appropriate. That'd be more testable, but more expensive and less clear. I'm happy to take that approach if you think it's worthwhile, though.

Reply:
Having written all that, I'm going to do it. :)